notify-api/src/notify_api/models/notification.py

```diff
@@ -74,6 +87,7 @@ class NotificationStatus(BaseEnum):
         """Enum for the Notification Status."""
 
         PENDING = auto()
+        SENT = auto()
         DELIVERED = auto()
```

Comment:
Is this status used anywhere now (potentially it was replaced by SENT)?

Reply:
Yes, when we send the request to email service we don't know it delivery or not. Until we receive the status from the callback service.